imls-windows-installer/build_wifi-hw-search.ps1

```diff
@@ -17,4 +22,10 @@ $exe_path = "$wd\wifi-hardware-search-windows.exe"
 # -- Move the exe into imls-windowsinstaller
 Write-Host "Executable built. Terminating script"
 Move-Item -Path $exe_path -Destination $installer_path
+
+# -- Update location
+Set-Location ..
+Set-Location ..
+Set-Location ..
+Set-Location imls-windows-installer
```

Comment:
How does this work if we run this script by itself? I'm a bit wary of changing the directory under the user's feet; do we come back to the original directory after this is done? (similar to how `pushd` and `popd` work in bash?)

Reply:
The user starts out in the imls-windows-installer directory where the script is. When building the executable, it moves to the imls-raspberry-pi sub-folders. I forgot to add the code at the end changing it back to the imls-windows-installer, which is why I added it in this commit.

Your comment made me realize I could do this in one line rather than 4, so I'll make that update and push those changes.